iotilebuild/test/test_iotilebuild/test_scons.py

```diff
@@ -108,3 +110,26 @@ def test_build_prerelease(tmpdir):
         assert err == 0
     finally:
         os.chdir(olddir)
+
+def test_bootstrap_file(tmpdir):
+    """Make sure we can create a bootstrap file"""
+    olddir = os.getcwd()
+    builddir = copy_folder('component_bootstrap', tmpdir)
+
+    try:
+        os.chdir(builddir)
+        err = subprocess.check_call(["iotile", "build"])
+        assert err == 0
+
+        print os.listdir(os.path.join('build','output',builddir))
+
+        hexdata = IntelHex(os.path.join('build','output', 'test_hexfile.hex'))
+        assert hexdata.segments() == [(0x10001014, 0x10001018)]
+        hexdata = IntelHex(os.path.join('build','output', 'test_elffile.hex'))
```

Comment:
You can remove this line once you refactor the build step to not leave temporary hex files.